internal/db/schema/migrations/20231214192507_add_refresh_service_to_link_table.sql

```diff
@@ -0,0 +1,11 @@
+-- +goose Up
```

Comment:
I'm missing the same migration for the `claims` table + the repository that saves the refresh service